cfme/middleware/provider/hawkular.py

```diff
@@ -156,6 +156,14 @@ def is_refreshed_db(self):
                                        ems.updated_on).filter(ems.name == self.name).first()
         return dates.updated_on > dates.created_on
 
+    @variable(alias='ui')
```

Comment:
this will eventually be sentaku-ized, but currently the variable/variant usage for this file is consistent so its ok 👍